shell/platform/darwin/macos/framework/Source/FlutterSurfaceManager.mm

```diff
@@ -29,7 +29,8 @@ @interface FlutterSurfaceManager () {
 @implementation FlutterSurfaceManager
 
 - (instancetype)initWithLayer:(CALayer*)containingLayer
-                openGLContext:(NSOpenGLContext*)openGLContext {
```

Comment:
IOSurfaceHolder like in my metal PR. FrameBufferProvider, which is injected into FlutterSurfaceManager and also FlutterGLCompositor